src/styles/blocks/card.scss

```diff
@@ -0,0 +1,85 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  background-color: #fff;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-decoration: none;
+    color: #060b35;
+    margin: 0 0 4px;
+  }
+
+  &__code {
+    font-family: inherit;
+    font-size: inherit;
+    font-weight: 400;
+    line-height: 14px;
+    color: #616070;
+    margin-bottom: 16px;
+    margin-top: 0;
+  }
+
+  &__stars {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+  }
+
+  .card__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    .card__price_title {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: #616070;
+    }
+
+    .card__price_value {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: #060b35;
+    }
+  }
+
+  &__buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    height: 40px;
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+    border: 1px solid #00acdc;
+    background-color: #00acdc;
+    border-radius: 5px;
+    color: #fff;
+    text-decoration: none;
+    text-transform: uppercase;
+  }
+
+  &__buy:hover {
+    background-color: #fff;
+    color: #00acdc;
+    border-color: #00acdc;
```

Comment:
Let's use variables for repeating properties.

Reply:
@seriuksergii please, fix @roman-mirzoian comment